pkg/store/acceptance_test.go

```diff
@@ -49,17 +48,6 @@ func testLabelAPIs(t *testing.T, startStore func(extLset labels.Labels, append f
 		labelNameCalls   []labelNameCallCase
 		labelValuesCalls []labelValuesCallCase
 	}{
-		{
-			desc: "no label in tsdb, empty results",
```

Comment:
We cannot have a bucket with empty blocks i think; at least i didnt manage to create one quickly